### DISCUSSION TITLE
Frage zur Abgabe

### DISCUSSION THREAD

**Discussion by Memla123** [irgroup-classrooms/dis08-2021#55] [Q&A]
Hallo Herr Schaer, 

Wir hatten eine Frage zur Abgabe vom Assignment 2 , nämlich im welchem Format wir die Aufgaben abgeben sollen?
 (als Jupiter Notebook oder als Markdown)
Vielen Dank im Voraus 
Viele Grüße 
Data deputy

**Comment by phschaer** [comment]
Ein Jupyter Notebook bietet sich an, da Sie hier Dokumentation und lauffähigen Code, Seite an Seite präsentieren können. Prinzipiell können Sie das auch gerne getrennt voneinander umsetzen. Das steht ihnen frei. Ich empfehle aber die Notebook-Variante.
